static/js/app.js

```diff
@@ -87,6 +87,14 @@ function untoast() {
   clearTimeout(toastTimeout);
 }
 
+function rename(name) {
+  if (!config.name) {
+    config.name = "";
+  }
```

Comment:
Defensive code to coerce name to a string if its a number or whatever -or- ritualized cargo cult prelude? You decide.